stt:
    default_param:
        model_file_path: "$(find harmoni_models)/stt/deepspeech-0.9.3-models.pbmm"
        scorer_path: "$(find harmoni_models)/stt/deepspeech-0.9.3-models.scorer"
        lm_alpha: 0.75 
        lm_beta: 1.85
        beam_width: 700
        t_wait: 0.5
        subscriber_id: "default"
